vendor/k8s.io/kubernetes/pkg/volume/glusterfs/glusterfs.go

```diff
@@ -476,9 +484,9 @@ func (r *glusterfsVolumeProvisioner) Provision() (*api.PersistentVolume, error)
 		return nil, err
 	}
 	r.provisioningConfig = *cfg
-
 	glog.V(4).Infof("glusterfs: creating volume with configuration %+v", r.provisioningConfig)
-	glusterfs, sizeGB, err := r.CreateVolume()
+	reqGid = gidMin + gidRandomizer.Int63n(gidMax)
```

Comment:
If we are dropping the random GID method, I dont know how we can keep track of the used or in use GIDs and create new one based on this.